scripts/build/debian/config.json

```diff
@@ -39,7 +39,8 @@
     "libxrandr2",
     "libxrender1",
     "libxss1",
-    "libxtst6"
+    "libxtst6",
+    "polkit-1-auth-agent | policykit-1-gnome"
```

Comment:
Yes. However, it was only created in 2016, so older OSes won't have that package. There are several packages that `Provides: policykit-1-gnome`, so that package name provides the same sort of virtual package function, so hopefully this will cover most cases. Although now that I am thinking about it, I should check to see if there is a KDE version as well.